translations/ru_RU.ts

```diff
@@ -273,15 +273,15 @@
     </message>
     <message>
         <source>Ctrl+]</source>
-        <translation>Ctrl+] (Ctrl + Ъ)</translation>
+        <translation>Ctrl+] (Ctrl+Ъ)</translation>
     </message>
     <message>
         <source>Unindent</source>
         <translation>Убрать отступ</translation>
     </message>
     <message>
         <source>Ctrl+[</source>
-        <translation>Ctrl+[ (Ctrl + Х)</translation>
+        <translation>Ctrl+[ (Ctrl+Х)</translation>
```

Comment:
These are not working.

![image](https://user-images.githubusercontent.com/30581822/87489918-da5a1a00-c676-11ea-8f7d-2e2881c1d757.png)